pkg/network/admitter/netiface.go

```diff
@@ -154,31 +156,23 @@ func validateInterfaceModel(field *k8sfield.Path, idx int, iface v1.Interface) [
 	return nil
 }
 
-func validateMacAddress(field *k8sfield.Path, idx int, iface v1.Interface) []metav1.StatusCause {
+func ValidateMacAddress(ifaceFieldPath, ifaceName, macAddress string) []metav1.StatusCause {
```

Comment:
@EdDev Since this change impact network codebase, I would prefer your feedback here

Reply:
@EdDev Could you give us some comments?

Reply:
The shared logic is:
```go
if macAddress != "" {
    mac, err := net.ParseMAC(macAddress)
    if err != nil {
        return fmt.Errorf("malformed MAC address (%s)", macAddress)
    }
    if len(mac) > 6 {
        return fmt.Errorf("too long MAC address (%s)", macAddress)
    }
}
```

Only this  logic needs to be shared, not to make the existing validation helper generic.
I.e. create a 3rd helper that performed the shared logic and use it in the existing validation function and in a new one for the clone.

I would prefer the existing validation function signature is not changed. It validates the VMI Spec, not a generic mac address with a different field.